swagger.json

```diff
@@ -2009,6 +2046,92 @@
           }
         }
       },
+      "patch": {
```

Comment:
Changing a web service. The type and job_id should not be changed (therefore removed), but the configuration parameters (args) might be needed to changed without creating a new service id. Otherwise every change would trigger a new URL for (shared) services.